src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
```

Comment:
Prefer using `const` over `let` for `totalCost` if you do not plan to reassign it, which in this case you do not (checklist item 3).